src/main/java/io/kuzzle/sdk/core/KuzzleDocument.java

```diff
@@ -151,42 +151,37 @@ public void onError(JSONObject error) {
   }
 
   /**
-   * Replaces the current content with the last version of this document stored in Kuzzle.
-   *
-   * @return the kuzzle document
+   * Gets a refreshed copy of the current object
    */
-  public KuzzleDocument refresh() {
-    return this.refresh(null, null);
+  public void refresh() {
+    this.refresh(null, null);
   }
 
   /**
-   * Refresh kuzzle document.
+   * Gets a refreshed copy of the current object
    *
    * @param options the options
-   * @return the kuzzle document
    */
-  public KuzzleDocument refresh(final KuzzleOptions options) {
-    return this.refresh(options, null);
+  public void refresh(final KuzzleOptions options) {
+    this.refresh(options, null);
   }
 
   /**
-   * Refresh kuzzle document.
+   * Gets a refreshed copy of the current object
    *
    * @param listener the listener
-   * @return the kuzzle document
    */
-  public KuzzleDocument refresh(final KuzzleResponseListener<KuzzleDocument> listener) {
-    return this.refresh(null, listener);
+  public void refresh(final KuzzleResponseListener<KuzzleDocument> listener) {
+    this.refresh(null, listener);
   }
 
   /**
-   * Replaces the current content with the last version of this document stored in Kuzzle.
+   * Gets a refreshed copy of the current object
    *
    * @param options  the options
    * @param listener the listener
-   * @return kuzzle document
    */
-  public KuzzleDocument refresh(final KuzzleOptions options, final KuzzleResponseListener<KuzzleDocument> listener) {
+  public void refresh(final KuzzleOptions options, final KuzzleResponseListener<KuzzleDocument> listener) {
```

Comment:
In the Javascript SDK and the SDK documentation versions implementing this change, I made the callback argument required instead of optional.
So the `listener` argument should now be annoted with `@NonNull`, and a check should be added, throwing an `IllegalArgumentsException` if no `listener` is provided.